shell/platform/windows/win32_window.cc

```diff
@@ -198,6 +265,35 @@ Win32Window::HandleMessage(UINT const message,
                 static_cast<double>(WHEEL_DELTA)),
                0.0);
       break;
+    case WM_INPUTLANGCHANGE:
+      // TODO(cbracken): pass this to TextInputManager to aid with language-specific issues.
+      break;
+    case WM_IME_SETCONTEXT:
+      OnImeSetContext(message, wparam, lparam);
+      result_lparam &= ~ISC_SHOWUICOMPOSITIONWINDOW;
```

Comment:
Please add a version of the PR description explanation of this as a comment here.

Reply:
Done.